src/main/java/uk/co/ramp/covid/simulation/place/Hospital.java

```diff
@@ -15,13 +15,17 @@
 
     public Hospital(Size s) {
         super(s);
+        // TODO: Adjustment or constant?
         transAdjustment = PopulationParameters.get().buildingProperties.hospitalTransmissionConstant;
```

Comment:
This looked suspicious to have adjustment being set to the constant parameter. I've emailed @paulbessell to determine if the parameter name is wrong, or the field we set is wrong!